sources/committees.js

```diff
@@ -2,47 +2,67 @@ const axios = require('axios'),
       cheerio = require('cheerio');
 
 const committees = {
-  getBase() {
-    return axios.get('https://www.parliament.uk/business/committees/committees-a-z/').then(async response => {
-      const $ = cheerio.load(response.data);
-      const allCommittees = [];
-
-      $('.a-to-z-listing .square-bullets-a-to-z li h3 a').each((item, element) => {
-        const href = $(element).attr('href');
-        const guid = href.includes('http') ? href : `https://www.parliament.uk${href}`;
-        allCommittees.push({
-          description: null,
-          enabled: 0,
-          guid,
-          last_updated: null,
-          rss_link: null,
-          title: $(element).text(),
-          type: 'committee'
-        });
-      });
+  /**
+   * Get the base list of committees
+   * @param  {string} url URL of the committees A-Z page
+   * @return {Promise}
+   */
+  async getBase(url) {
+    const response = await axios.get(url);
 
-      for(let item in allCommittees) {
-        let rss_link = await this.getCmsPageInstanceId(allCommittees[item].guid);
-        if(rss_link) {
-          rss_link = `https://www.parliament.uk/g/rss/committee-feed/?type=Committee_Detail_Mixed&pageInstanceId=${rss_link}`;
-          allCommittees[item].rss_link = rss_link;
-          allCommittees[item].enabled = 1;
-        }
-      }
+    const $ = cheerio.load(response.data);
+    const committees = [];
 
-      return allCommittees;
+    $('.a-to-z-listing .square-bullets-a-to-z li h3 a').each((item, element) => {
```

Comment:
Would be better to avoid excessive specificity, flatter DOM helps improve selector performance, would also be useful to comment the url for context